content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Manage API Server load with the Flow Control API
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
```

Comment:
:ok_hand:

Reply:
Oops, wrong button, and GitHub no longer has my work-in-progress :disappointed:

Reply:
Actually this metadata is _I think_ mainly for task and tutorial pages. It seems OK to leave it in.

Reply:
I'll leave it in for now.